src/CppParser/Parser.cpp

```diff
@@ -3832,7 +3832,7 @@ ParserResult* Parser::ParseLibrary(const std::string& File, ParserResult* res)
     }
 
     auto BinaryOrErr = llvm::object::createBinary(FileEntry);
-    if (BinaryOrErr.getError())
+    if (!BinaryOrErr) // false indicates error
```

Comment:
Please remove the comment, it's clear enough without it and it makes the line even longer compared to if takeError() was used instead.